tests/tool/Integration.Tests/IntegrationScenarios/ConsoleSample/csharp/Upgraded/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
```

Comment:
I don't think we need an integration test for this. Unit testing should be sufficient.